airflow/models/taskinstance.py

```diff
@@ -1184,7 +1184,7 @@ def signal_handler(signum, frame):  # pylint: disable=unused-argument
         task_copy.post_execute(context=context, result=result)
 
         end_time = time.time()
-        duration = end_time - start_time
+        duration = timedelta(seconds=end_time - start_time)
```

Comment:
Isn't that microseconds ?

Reply:
No, `time.time()` is a float value representing seconds, see the [documentation](https://docs.python.org/3/library/time.html#time.time).